### ISSUE TITLE
I am unsure what I need to do and I certainly don't know how to do it.

### ISSUE THREAD

**Issue by Matt-The-Eternal-Champion** [MomentaiChris26/uac-accessbility-week-challenge#1]
If instructions can be in plain english for someone with no coding experience, that would be very helpful for my use case.

**Comment by MomentaiChris26** [comment]
Hi @Matt-The-Eternal-Champion! Happy to provide some instructions for you! 

I've made it a little easier to get the code up and running without needing to install additional software. 

The goal is to modify a file called index.html to reach an accessibility score of 100. the score represents how accessibility friendly the page is. 

**Tools needed - Google lighthouse**
There is a tool called Google lighthouse which can be found in the browser that will give you that score. 
It looks like this in the browser. To access it right click anywhere on the page, and select "inspect". This will bring up a panel on the right with some tabs. You can select the "lighthouse" tab which will bring up something similar to the screenshot below. 
![image](https://user-images.githubusercontent.com/45407977/199639028-ecaafbb7-f994-4111-a10d-a68b827fdd5c.png)

Select the options as per the screenshot and press "Analyze Page Load", give it a few moments and it will return an accessibility score. 

**To get started**
_To do this challenge you'll need to do a little coding but I have included the solution for you to compare._ 
1. I have attached a link for you to take a look at. https://codesandbox.io/s/blissful-phoebe-ig7059?file=/index.html
2. There you'll find a file called index.html and there is a solution version there for you to take a look at. 
3. To check the accessibility score on the index.html do the following:
  - Open the link above, and click on the icon as shown in the screenshot. 
  - This will open up a new window. From there run the instructions to open google lighthouse and run the Analyze Page 
  Load to view the score. 
  - Under the score will be a list of "accessibility issues" which you can fix by modifying the index.html. If you're stuck take 
  a look at the solution to get some hints

Hope that helps! And good luck!

**Comment by Matt-The-Eternal-Champion** [comment]
Thank you :)
